tools/config/seed.config.ts

```diff
@@ -285,136 +479,166 @@ export class SeedConfig {
+    return this.SYSTEM_CONFIG_DEV;
+  }
+  set SYSTEM_CONFIG(val: any) {
+    warnInvalidSet('SYSTEM_CONFIG');
+  }
 
   /**
    * The system builder configuration of the application.
    * @type {any}
    */
-  SYSTEM_BUILDER_CONFIG: any = {
-    defaultJSExtensions: true,
-    packageConfigPaths: [
-      join(this.PROJECT_ROOT, 'node_modules', '*', 'package.json'),
-      join(this.PROJECT_ROOT, 'node_modules', '@angular', '*', 'package.json')
-    ],
-    paths: {
-      [`${this.TMP_DIR}/*`]: `${this.TMP_DIR}/*`,
-      '*': 'node_modules/*'
-    },
-    packages: {
-      '@angular/core': {
-        main: 'index.js',
-        defaultExtension: 'js'
-      },
-      '@angular/compiler': {
-        main: 'index.js',
-        defaultExtension: 'js'
-      },
-      '@angular/common': {
-        main: 'index.js',
-        defaultExtension: 'js'
-      },
-      '@angular/http': {
-        main: 'index.js',
-        defaultExtension: 'js'
+  get SYSTEM_BUILDER_CONFIG(): any {
+    return {
+      defaultJSExtensions: true,
+      packageConfigPaths: [
+        join(this.PROJECT_ROOT, 'node_modules', '*', 'package.json'),
+        join(this.PROJECT_ROOT, 'node_modules', '@angular', '*', 'package.json')
+      ],
+      paths: {
+        [`${this.TMP_DIR}/*`]: `${this.TMP_DIR}/*`,
+        '*': 'node_modules/*'
       },
-      '@angular/platform-browser': {
-        main: 'index.js',
-        defaultExtension: 'js'
-      },
-      '@angular/platform-browser-dynamic': {
-        main: 'index.js',
-        defaultExtension: 'js'
-      },
-      '@angular/router': {
-        main: 'index.js',
-        defaultExtension: 'js'
-      },
-      'rxjs': {
-        defaultExtension: 'js'
+      packages: {
+        '@angular/core': {
+          main: 'index.js',
+          defaultExtension: 'js'
+        },
+        '@angular/compiler': {
+          main: 'index.js',
+          defaultExtension: 'js'
+        },
+        '@angular/common': {
+          main: 'index.js',
+          defaultExtension: 'js'
+        },
+        '@angular/http': {
+          main: 'index.js',
+          defaultExtension: 'js'
+        },
+        '@angular/platform-browser': {
+          main: 'index.js',
+          defaultExtension: 'js'
+        },
+        '@angular/platform-browser-dynamic': {
+          main: 'index.js',
+          defaultExtension: 'js'
+        },
+        '@angular/router': {
+          main: 'index.js',
+          defaultExtension: 'js'
+        },
+        'rxjs': {
+          defaultExtension: 'js'
+        }
       }
-    }
-  };
+    };
+  }
+  set SYSTEM_BUILDER_CONFIG(val: any) {
+    warnInvalidSet('SYSTEM_BUILDER_CONFIG');
+  }
 
   /**
    * The Autoprefixer configuration for the application.
    * @type {Array}
    */
-  BROWSER_LIST = [
-    'ie >= 10',
-    'ie_mob >= 10',
-    'ff >= 30',
-    'chrome >= 34',
-    'safari >= 7',
-    'opera >= 23',
-    'ios >= 7',
-    'android >= 4.4',
-    'bb >= 10'
-  ];
+  get BROWSER_LIST() {
```

Comment:
I think this is fine. We don't have implicit any and the compiler can easily perform type inference.